dataservice/api/person/resources.py

```diff
@@ -97,14 +97,12 @@ def delete(self, kf_id):
 
         db.session.delete(person)
         db.session.commit()
-        return {'status': 200,
-                'message': 'person deleted',
-                'content': {'persons': [person]}}, 200
+
+        return kf_response(person, 200, 'person deleted')
 
     def _not_found(self, kf_id):
```

Comment:
One test is failing: test_get_not_found in tests > person > test_resources.  

Change this method to the following to fix:
```python
    def _not_found(self, kf_id):
        """
        Temporary helper - will do error handling better later
        """
        return kf_response(code=404,
                           message='person with kf_id \'{}\' not found'.
                           format(kf_id))
```
Since the kf_response method already returns the tuple: (result dict, status code), we can just return the result of kf_response.